app/screens/sso/sso_authentication.tsx

```diff
@@ -87,9 +90,10 @@ const SSOAuthentication = ({doSSOLogin, loginError, loginUrl, serverUrl, setLogi
             const bearerToken = resultUrl.query?.MMAUTHTOKEN;
             const csrfToken = resultUrl.query?.MMCSRF;
             if (bearerToken && csrfToken) {
+                setLoginSuccess(true);
                 doSSOLogin(bearerToken, csrfToken);
             }
-        } else if (Platform.OS === 'ios' || result.type === 'dismiss') {
+        } else if (Platform.OS === 'ios' && result.type === 'dismiss') {
```

Comment:
@enahum if on Android, and user "dismiss" or (x) out of Okta sso, would this if logic change miss that user?

Reply:
@rahimrahman yes thank you, this was unintended.